packages/aws-cdk/lib/init.ts

```diff
@@ -194,12 +193,18 @@ interface ProjectInfo {
   readonly name: string;
 }
 
+const versionedTemplatesDir: Promise<string> =
+  new Promise(async resolve => {
+    const majorVersion = versionNumber().replace(/\..+/, '');
+    resolve(path.join(__dirname, 'init-templates', `v${majorVersion}`));
+  });
```

Comment:
make this a function that returns a promise instead?

Reply:
Sure. I did it this way to match the other similar functions (in context), but no reason this needs to be a const like this.